test/fixers/test_sqlformat_fixer_callback.vader

```diff
@@ -0,0 +1,24 @@
+Before:
+  Save g:ale_sql_sqlformat_executable
+  Save g:ale_sql_sqlformat_options
+
+After:
+  Restore
+
+Execute(The sqlformat callback should return the correct default values):
+  AssertEqual
+  \ {
+  \   'command': ale#path#BufferCdString(bufnr('')) . ale#Escape('sqlformat') . ' -'
```

Comment:
Don't forget to fix this and the other test if you remove `ale#path#BufferCdString()` from elsewhere.

Reply:
Thank you